src/components/Alert/Alert.vue

```diff
@@ -25,16 +47,16 @@ export default {
   },
   computed: {
     info () {
-      return this.variant === 'info';
+      return this.variant === 'info' ? 'info' : false;
```

Comment:
wondering why you added the : false part ?

Reply:
same, previously `info` returned a boolean, now sometimes a string sometimes a boolean?

Reply:
I agree they aren't needed. Thanks for the feedback!